.github/workflows/pull-request-closed.yml

```diff
@@ -1,29 +1,29 @@
 name: Close pull request
 on:
-  pull_request:
-    types:
-      - closed
+    pull_request:
```

Comment:
Not sure if the YAML spacing changes are intentional, but I think 2 spaces is more common. Suggest adding a `.editorconfig` or similar to help keep this consistent across team members.